torch/lib/TH/generic/THTensorMath.c

```diff
@@ -2927,7 +2931,7 @@ void THTensor_(pow)(THTensor *r_, THTensor *t, real value)
     THTensor_(cmul)(r_, t, t);
   }
   else if(value == 3){
-    TH_TENSOR_APPLY2(real, t, real, r_, *r__data = *t_data * *t_data * *t_data;);
+    TH_TENSOR_APPLY2(real, r_, real, t, *r__data = *t_data * *t_data * *t_data;);
```

Comment:
Damn copy-paste, sorry for the introduced bug... :-/

Reply:
Just checked this again in more detail, this was not a bug, because the order of the arguments doesn't matter (phew! :D).
But I suppose this ordering looks nicer :-)

Reply:
Yup, I just changed it to be more consistent with the surrounding code! 😊